### PR TITLE
My Activities Page

### PR DESCRIPTION
**Pull request by liviavinci**

### CODE REVIEW

**Comment by liviavinci** [comment]
screen shot 2018-03-07 at 9 43 05 pm

File: src/scenes/Activities/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    const { classes } = this.props;
+    const { tabIdx } = this.state;
+
+    return (
+      <div>
+        <Tabs indicatorColor="primary" value={tabIdx} onChange={this.handleTabChange} className={classes.activitiesTabWrapper}>
+          <Tab label="Result Setting" />
+          <Tab label="Finalize" />
+          <Tab label="Withdraw" />
+          <Tab label="Activities History" />
```
Comment (dwalintukan) [inline src/scenes/Activities/index.js]: convert all the labels to use localization, `intl.formatMessage()`

File: src/scenes/Activities/styles.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const styles = (theme) => ({
+  activitiesTabWrapper: {
+    background: 'white',
```
Comment (dwalintukan) [inline src/scenes/Activities/styles.js]: should we use `theme.palette.background.paper`? or is that specifically for Paper components?

File: src/scenes/Dashboard/vote.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+/* eslint react/no-array-index-key: 0, no-nested-ternary:0 */ // Disable "Do not use Array index in keys" for options since they dont have unique identifier
+
+import React, { PropTypes } from 'react';
```
Comment (dwalintukan) [inline src/scenes/Dashboard/vote.js]: separate proptypes import `import PropTypes from ...`

File: src/components/EventsEmptyBg/index.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
```
Comment (dwalintukan) [inline src/components/EventsEmptyBg/index.js]: separate PropTypes import.

File: src/components/EventCardsGridContainer/index.js [hunk 5]
```diff
@@ -0,0 +1,316 @@
+/* eslint react/no-array-index-key: 0, no-nested-ternary:0 */ // Disable "Do not use Array index in keys" for options since they dont have unique identifier
+
+import React, { PropTypes } from 'react';
```
Comment (dwalintukan) [inline src/components/EventCardsGridContainer/index.js]: separate proptypes import

File: src/components/NavBar/components/NavEventsButtons/index.js [hunk 6]
```diff
@@ -1,73 +1,52 @@
...
-
-    if (window.location.pathname !== '/') {
-      window.location.assign('/');
-    }
-  }
-
   render() {
     const { classes } = this.props;
+    console.log('this.props');
+    console.log(this.props);
```
Comment (dwalintukan) [inline src/components/NavBar/components/NavEventsButtons/index.js]: remove logs

File: src/components/EventCardsGridContainer/index.js [hunk 7]
```diff
@@ -0,0 +1,316 @@
...
+
+        break;
+      }
+      default: {
+        throw new RangeError(`Invalid tab position ${eventStatusIndex}`);
+      }
+    }
+
+    return (
+      <Grid container spacing={theme.padding.sm.value}>
```
[the remark this answers is not in the document]
Comment (liviavinci) [reply src/components/EventCardsGridContainer/index.js]: not really, this is a material component property

**Comment by dwalintukan** [comment]
:shipit:

**Pull request merged**